master/docs/examples/hello.cfg

```diff
@@ -66,8 +62,9 @@ c['titleURL'] = "http://www.hello.example.com/"
 c['buildbotURL'] = "http://localhost:8080"
 
 c['slavePortnum'] = 8007
-c['manhole'] = master.Manhole(9900, "username", "password")
+c['manhole'] = util.PasswordManhole(9900, "username", "password")
 
-c['status'] = [html.WebStatus(http_port=8080),
-               ]
+# This is not available in nine
```

Comment:
you can declare a basic www instead of just commenting.

c['www'] = {
    'port': 8080
}